docs/openshift_ci.md

```diff
@@ -12,7 +12,50 @@ Prior to opening a PR it is useful and efficient to be able to run tests locally
 
 `openshift-tests` requires a kubeconfig and `oc` binary as well. MicroShift generates a kubeconfig located in `/var/lib/microshift/resources/kubeadmin/kubeconfig`, but we will see how to use it depending on local vs remote access to MicroShift instances.
 
-Once we have the kubeconfig working, we can launch the tests. In order to run `openshift-tests` for `MicroShift` we need to provide KUBECONFIG and one specific option:
+Once we have the kubeconfig working, we need to set the static manifests pointed through `STATIC_CONFIG_MANIFEST_DIR` environment variable.
+
+```sh
+mkdir ~/manifests
+export STATIC_CONFIG_MANIFEST_DIR=~/manifests
+```
+
+Currently only manifests from `[infrastructure|network].config.openshift.io/v1` are supported as static manifests.
+
+Exemplary manifests:
```

Comment:
Since these manifests are required for the normal test runtime, can we make the yaml snapshots into commands to create the files? Something like the following:
```
MICROSHIFT_URL=<microshift-server-url>
cat > ${STATIC_CONFIG_MANIFEST_DIR}/single-replica.yaml <<EOF
...
...
status:
  apiServerURL: ${MICROSHIFT_URL}
...
EOF

Reply:
Thanks for the suggestion. Added.